src/TiciTacaToeyGameEngine.ts

```diff
@@ -322,7 +356,7 @@ class TiciTacaToeyGameEngine implements GameEngine {
             },
           };
         }
-
+        game.timers[nextPlayer].start();
```

Comment:
The timer of the nextPlayer should only be started, if game has not ended in victory or tie - so check for those conditions